viz/cmd/install_test.go

```diff
@@ -86,13 +85,6 @@ func TestRender(t *testing.T) {
 			},
 			"install_default_overrides.golden",
 		},
-		{
-			map[string]interface{}{
-				"grafana":    map[string]interface{}{"enabled": false},
-				"grafanaUrl": "external-grafana.com",
```

Comment:
nit: probably have a \`grafana.url\` set just as sanity check for this test? :thinking:

Reply:
Good catch as well :+1:

Reply:
Sure thing :+1: